spotbugs-exclude.xml

```diff
@@ -8,6 +8,22 @@
        https://spotbugs.readthedocs.io/en/latest/bugDescriptions.html
   -->
 <FindBugsFilter>
+    <!-- These open matches are new rules that we have not yet addressed. Let's block them generically, and come
```

Comment:
@jeromevdl i'm not sure why spotbugs suddenly has more rules, but it does. As the purpose of this is to just get V2 snapshots + docs publishing i've excluded them generically (e.g. not bound to a particular class or field) for now.

Reply:
ok